plugins/vbe/fit_test.c

```diff
@@ -0,0 +1,645 @@
+/* -*- Mode: C; tab-width: 8; indent-tabs-mode: t; c-basic-offset: 8 -*-
+ *
+ * Tests for libfit
+ *
+ * Copyright (C) 2022 Google LLC
+ * Written by Simon Glass <sjg@chromium.org>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <glib-2.0/glib.h>
+#include <libfdt.h>
+#include <stdio.h>
+
+#include "fit.h"
+#include "fit_test.h"
```

Comment:
don't think you need this include (if you axe that file)

Reply:
done